docs/specification/1.0.md

```diff
@@ -452,7 +452,7 @@ Field | Optionality | Type | Description
 `indicator` | REQUIRED | *string* | Urgency/importance of what this card conveys. Allowed values, in order of increasing urgency, are: `info`, `warning`, `critical`. The CDS Client MAY use this field to help make UI display decisions such as sort order or coloring.
 `source` | REQUIRED | *object* | Grouping structure for the **Source** of the information displayed on this card. The source should be the primary source of guidance for the decision support the card represents.
 <nobr>`suggestions`</nobr> | OPTIONAL | *array* of **Suggestions** | Allows a service to suggest a set of changes in the context of the current activity (e.g.  changing the dose of the medication currently being prescribed, for the `medication-prescribe` activity). If suggestions are present, `selectionBehavior` MUST also be provided.
-`selectionBehavior` | OPTIONAL | *string* | Describes the intended selection behavior of the suggestions in the card. Currently, the only allowed value is `at-most-one`, indicating that the user may choose none or at most one of the suggestions. Future versions of the specification may expand this behavior, so CDS Clients that do not understand the value MUST treat the card as an error. CDS Clients MUST support the value of `at-most-one`.
+`selectionBehavior` | OPTIONAL | *string* | Describes the intended selection behavior of the suggestions in the card. Allowed values are: `at-most-one`, indicating that the user may choose none or at most one of the suggestions;`any`, indicating that the end user may choose any number of suggestions including none of them and all of them. CDS Clients that do not understand the value MUST treat the card as an error.
```

Comment:
Any concerns with removing a 1.0-spec requirement that 'at-most-one' be supported?  I understand it was required as being the only option, but trying to think through any potential issues with removing something that was stated as a requirement.

Reply:
I'm in fine with the above wording. Do I need to do anything now - I don't see an "accept changes" button or anything.

Reply:
Hey guys,

`selectionBehavior` is OPTIONAL. Making is REQUIRED would be a breaking change. If no selectionBehavior is provided, how does a client know what behavior to enact?

Isaac

Reply:
Our call on 11/13 walked through my initial comment and I think we clarified.

Not sure I understand your comment, Isaac, on the optionality of selectionBehavior and "If no selectionBehavior is provided, how does a client know what behavior to enact?"  I know you're familiar with the [conditionally required](https://github.com/cds-hooks/docs/issues/509) aspect of selectionBehavior, so can you clarify?